simulators/mujoco_simulator/revolve2/simulators/mujoco_simulator/_scene_to_model.py

```diff
@@ -114,8 +114,8 @@ def scene_to_model(
         # Add actuation to joints
         _add_joint_actuators(joints_and_names, multi_body_system_mjcf)
 
-        # Add Sensors
-        _add_sensors(rigid_bodies_and_names, mbs_i, multi_body_system_mjcf, env_mjcf)
+        # Add sensors and motors
```

Comment:
this should be two seperate functions.
Generally we always want functions that only serve one purpose
lookup "Single Responsibility Principle"

Reply:
My problem was, that the joints are defined in the URDF, and then actuation is added to them. Similarly, for motors, the site needs to be defined and the actuation is attached to it. However, I haven't found a way of defining the sites in the URDF. The sensors have to also do it this way, hence why I used that method. If we were to make two separate functions we would have to iterate through the rigid_bodies twice, which is why for now I kept it in one.

There should be a solution that keeps the motors separate from the sensors. I'm just not quite sure what it is yet.